src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -1822,6 +1822,8 @@ export class VolumesListComponent extends EntityTableComponent implements OnInit
   protected editDatasetFormComponent: DatasetFormComponent;
   protected aroute: ActivatedRoute;
   private refreshTableSubscription: any;
+  private datasetQuery = 'pool.dataset.query';
+  private datasetQueryOptions = [[], {"extra": {"properties": ["type", "used", "available", "compression", "readonly", "dedup", "org.freenas:description", "compressratio"]}}]
```

Comment:
What's "org.freenas:description" representing? Couldn't a reference to it in API docs.

Reply:
It's used for showing comments of a dataset.
https://github.com/truenas/freenas/blob/9187e5a0d9ce4a69932aaf73e3464eac97dac3d9/src/middlewared/middlewared/plugins/pool.py#L2810